core/sql/sqlcomp/CmpSeabaseDDLtable.cpp

```diff
@@ -1181,6 +1421,30 @@ short CmpSeabaseDDL::createSeabaseTable2(
       return -1;
     }
 
+  // If creating an external table, go perform operation
+  if (createTableNode->isExternal())
+    {
+      retcode = createSeabaseTableExternalHive(cliInterface, createTableNode, currCatName, currSchName);
```

Comment:
There is definitively an issue here that was cut and pasted from create table like further down. 

First off, the createSeabaseExternalHive should only be returning 0 or -1.  0 is successful and -1 is failure.  A -2 is only needed if the associated HBase table needs to be removed (hence a cleanup).  Since there is no external table, a -2 should not be returned.  

This particular code code should return the retcode instead of -1:

  if (createTableNode->isExternal())
    {
      retcode = createSeabaseTableExternalHive(cliInterface, createTableNode, currCatName, currSchName);
      . . .
      return retcode;
    }